packages/chart/src/CdcChartComponent.tsx

```diff
@@ -593,25 +542,9 @@ const CdcChart = ({
       setSeriesHighlight(newConfig.legend?.seriesHighlight)
     }
 
-    setConfig(newConfig)
-  }
+    dispatch({ type: 'SET_CONFIG', payload: newConfig })
 
-  // Sorts data series for horizontal bar charts
-  const sortData = (a, b) => {
-    let sortKey =
-      config.visualizationType === 'Bar' && config.visualizationSubType === 'horizontal'
-        ? config.xAxis.dataKey
-        : config.yAxis.sortKey
-    let aData = parseFloat(a[sortKey])
-    let bData = parseFloat(b[sortKey])
-
-    if (aData < bData) {
-      return config.sortData === 'ascending' ? 1 : -1
-    } else if (aData > bData) {
-      return config.sortData === 'ascending' ? -1 : 1
-    } else {
-      return 0
-    }
+    setConfig(newConfig)
```

Comment:
If you're setting the config using a dispatch action shouldn't this be removed?

Reply:
Since setConfig is used everywhere in Core,Filters and all over charts i did this approach  const setConfig = newConfig => {
    dispatch({ type: 'SET_CONFIG', payload: newConfig })
  } so we dont have to update setConfig into   dispatch({ type: 'SET_CONFIG', payload: newConfig }) all over charts or data table